test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTest.java

```diff
@@ -53,4 +50,32 @@ public void upgradeTest() throws Throwable
                        row(1, 3, 3));
         }).run();
     }
+
+    @Test
+    public void reserializationDuringUpgradeFrom30() throws Throwable
```

Comment:
this test is a dup of the previous test, the only diff is it only upgrades a single node rather than one at a time.  Can we remove this?  The other test replicated the issue being fixed so don't think we need this.